docs/interactions/Message_Components.md

```diff
@@ -1,14 +1,38 @@
 # Message Components
 
-Message components--we'll call them "components" moving forward--are a framework for adding interactive elements to the messages your app or bot sends. They're accessible, customizable, and easy to use.
+Message components&#x2014;we'll call them "components" moving forward&#x2014;are a framework for adding interactive elements to the messages your app or bot sends. They're accessible, customizable, and easy to use.
```

Comment:
I replaced -- with — (em dash). The markdown should be able to render this html entity properly, but if it doesn't, I can change that.

Reply:
This won't render properly, can you use the unicode char?